docs/manage/fleet/change-network.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Connect a machine to a different network"
+linkTitle: "Connect to different network"
+weight: 55
+type: "docs"
+description: "Connect a machine to a different WiFi network."
+date: "2025-10-07"
+---
+
+This page will guide you to connect your machine to a different WiFi network.
+
+Should your machine be able to connect to multiple networks?
+
+- If **yes**, see [**Configure Networks**](/manage/fleet/system-settings/#configure-networks).
+- If **no**, continue on this page.
+
+Can your machine still connect to the old network?
+
+- If **yes**, see [Force provisioning mode](#force-provisioning-mode)
+- If **no**, see [Connect to a different WiFi network](#connect-to-a-different-wifi-network)
```

Comment:
at the risk of being a little harsh, i don't really like this and I also feel like it is a good example of why our docs are so impenetrable.  I have no idea what the first question is even asking.  and the second question seems unnecessary given it just links to the section at thbottom.  we should just get straight to the steps.